payload-test.json

```diff
@@ -0,0 +1,114 @@
+{
+    "name": "AWS-Analytical-Dataset-Generation",
+    "releaseLabel": "emr-5.24.1",
+    "serviceRole": "analytical_dataset_generator",
```

Comment:
This should be `emr_service_role`; IAM's a bit "funky" on EMR - there's a role that the EMR service needs to assume, which is this one. Then there's the role that the instances themselves need to have in order to carry out the business-logic in the script. So the `jobFlowRole` is the one that will have the KMS, S3, etc. permissions in it.

Reply:
As much as I want to defer to your knowledge, this was creating working clusters on Friday.  I thoguht we'd covered the service_role in the new roles we made in resource.tf